content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+   application on a One-App-Free database will trigger a 15-day trial. At the end of the trial, if a
+   paid subscription has not been added to the database, it will no longer be accessible.*
+
+Issue a refund with a credit note
+=================================
+
+A *credit note* is a document issued to a customer informing them that they have been credited a
+certain amount of money. They can be used to provide a full refund to a customer, or to adjust any
+remaining amount due. While they are usually created through the :menuselection:`Accounting` or
+:menuselection:`Invoicing` applications, they can be created through a :menuselection:`Helpdesk`
+:guilabel:`Ticket`.
+
+.. note::
+   Invoices must be posted before a credit note can be generated.
+
+To create a credit note from a :menuselection:`Helpdesk` :guilabel:`Ticket`, click on the
+:guilabel:`Refund` button in the upper left corner. Select the corresponding invoice from the
+:guilabel:`Invoices to Refund` drop-down.
+
+.. image:: after_sales/after-sales-refund-details.png
+   :align: center
+   :alt: View of a refund creation page.
+
+Choose a :guilabel:`Credit Method` from one of the following options:
+
+- :guilabel:`Partial Refund`: The credit note is created in draft and can be edited before being
+  issued.
+- :guilabel:`Full Refund`: The credit note is auto-validated and reconciled with the invoice. *This
+  is the option to choose if a validated invoice needs to be canceled*.
+- :guilabel:`Full refund and new draft invoice`: The credit note is auto-validated and reconciled
+  with the invoice. The original invoice is duplicated as a new draft. *This is the option to choose
+  if a validated invoice needs to be modified.*
+
+.. important::
+   The :guilabel:`Credit Method` options will *not* be available for invoices that have already been
+   paid.
+
+Make any necessary changes to the details of the credit note and click :guilabel:`Reverse.` Then
+click :guilabel:`Confirm` to post the credit note.
+
+.. image:: after_sales/after-sales-draft-credit.png
+   :align: center
+   :alt: View of a draft credit note.
+
+Once the credit note has been posted, a :guilabel:`Credit Notes` smart button will be added to the
+:menuselection:`Helpdesk` :guilabel:`Ticket`.
+
+.. image:: after_sales/after-sales-credit-note-smart-button.png
+   :align: center
+   :alt: View of smart buttons on a ticket focusing on the credit note button.
+
+.. seealso::
+   - :doc:`/applications/finance/accounting/receivables/customer_invoices/credit_notes`
+
+Generate coupons from a ticket
+==============================
+
+Coupons can be used to alter the price of products or orders. The usage constraints of a coupon are
+defined by conditional rules. *Coupon Programs* are configured in the :menuselection:`Sales` or
+:menuselection:`Website` applications.
+
+.. note::
+   The :menuselection:`eCommerce` module must be installed in order to create coupon codes from the
+   :menuselection:`Website` application.
```

Comment:
menuselections --> italics
we're not actively instructing users to go here (it's a hypothetical "if")